portal-impl/src/com/liferay/portal/service/base/CompanyLocalServiceBaseImpl.java

```diff
@@ -1272,6 +1284,12 @@ protected void runSQL(String sql) {
 	@BeanReference(type = RoleFinder.class)
 	protected RoleFinder roleFinder;
 
+	@BeanReference(
+		type = com.liferay.portal.kernel.service.SystemEventLocalService.class
+	)
+	protected com.liferay.portal.kernel.service.SystemEventLocalService
+		systemEventLocalService;
+
```

Comment:
@ChrisKian these are going to get blown out on the next `ant build-service-portal`.  This kind of reference needs to be added to the `service.xml` like so: https://github.com/liferay/liferay-portal/blob/48cd71b35a2d3b66e88f47685be7186cb7c52075/portal-impl/src/com/liferay/portal/service.xml#L189